sceptre/synapsedev/templates/accounts.yaml

```diff
@@ -71,6 +71,16 @@ Resources:
       LoginProfile:
         Password: !Ref InitNewUserPassword
         PasswordResetRequired: true
+  AWSIAMLukeMoenningUser:
+    Type: 'AWS::IAM::User'
+    Properties:
+      UserName: luke.moenning@sagebase.org
+      Groups:
+        - !Ref AWSIAMAllUsersGroup
+        - !Ref AWSIAMDeveloperUsersGroup
+      LoginProfile:
```

Comment:
Please remove the LoginProfile section (this is not needed anymore as you will use your JumpCloud user to get to the console).